python/lsst/ts/xml/data/sal_interfaces/ScriptQueue/ScriptQueue_Events.xml

```diff
@@ -85,6 +85,13 @@
       <Units>degree</Units>
       <Count>2</Count>
     </item>
+    <item>
+      <EFDB_Name>startTime</EFDB_Name>
+      <Description>The expected start time for this visit (TAI).</Description>
```

Comment:
Quick question: is this the estimated start of the _visit_, or of the _script_ (as stated in the commit message)? Hopefully the difference is never large enough to matter, but it would be good to be precise just in case.

Reply:
it will be of the visit... sorry, the commit message is wrong.. :(